src/main/java/billgates/use_cases/alter_entry/AlterEntryUseCase.java

```diff
@@ -17,33 +14,9 @@ public AlterEntryUseCase(DatabaseGateway gateway) {
     @Override
     public void alterEntry(int entryID, Object newValue, String alterColumn) {
```

Comment:
I think I have included 2 different versions of `alterEntry`, so it would be better to include both of them into the AlterEntryUseCase as `@Overridden` methods.

Reply:
solved by adding another method